app/pages/lab/visibility.cjsx

```diff
@@ -27,8 +31,22 @@ module.exports = React.createClass
   setterProperty: 'project'
 
   componentDidMount: ->
+    page = if @props.location?.query?.page? then @props.location.query.page else 1
+    @getWorkflowList page
+
+  onPageChange: (page) ->
+    nextQuery = Object.assign {}, @props.location.query, { page }
+    @context.router.push
+      pathname: @props.location.pathname
+      query: nextQuery
+    @getWorkflowList page
+    workflowList = document.getElementById("workflow-list")
```

Comment:
@jelliotartz `ref={node => this.workflowList = node` runs the callback when the component first renders. `this.workflowList` holds a reference to the DOM node, if it's a HTML element, or the component, if it's a React component. You don't need to use `ReactDOM` to access nodes from refs to HTML elements, you can just use `this.workflowList.scrollIntoView()`. 

I'm on my phone, so there may be syntax errors in that answer.